index.html

```diff
@@ -5,11 +5,152 @@
+            <a class="a-header-social" href="">English</a>
+            <a class="a-header-social" href="">My Page</a>
+            <a class="a-header-social" href="">Logout</a>
+        </div>
+        <header class="header">
+            <div class="logo-div">
+                <img src="images/logo.png" alt="logo-header" class="logo-img">
+            </div>
+            <section class="a-header-div" id="desk-buttons">
+                <a href="javascript:void(0)" id="clsebtn" class="closebtn">X</a>
+                <div class="header-ul">
+                    <a class="a-buttons a-home" href="index.html">Home</a>
+                    <a class="a-buttons" href="about.html">About</a>
+                    <a class="a-buttons" href="#">Program</a>
+                    <a class="a-buttons" href="#">Join</a>
+                    <a class="a-buttons" href="#">Sponsor</a>
+                    <a class="a-buttons" href="#">News</a>
+                    <button class="ccc-button">CC Campaign</button>
+                </div>
+            </section>
+        </header>
+    </div>
+
+<!-- -----------------------COVER--------------------------------- -->
+
+    <div class="intro-mobile">
+        <div class="hamburguer">
+            <span id="open-one" class="toggle-nav">&#9776;</span>
+        </div>
+        <div class="main-title">
+            <p class="title-paragraph">Hello Rockers in the world!</p>
+            <h1 class="title">VPN LIVE IN CONCERT WORLD TOUR 2022</h1>
+        </div>
+        <div class="paragraph">
+            <p class="paragraph-text">Join us for live music, food trucks and much more fun for all the family. Featuring one of the most popular bands in Quito, Ecuador VPN!. See you there and lets rock!</p>
+        </div>
+        <div class="date-place">
+            <p class="date">2022.02.10(THU) ~ 11(FRI)</p>
+            <p class="place">@Coliseo General Rumiñahui, Quito EC</p>
+        </div>
+    </div>
+    <div class="intro-desk">
+        <div class="main-title">
+            <p class="title-paragraph">Hello Rockers in the world!</p>
+            <h1 class="title">VPN LIVE IN CONCERT WORLD TOUR 2022</h1>
+        </div>
+        <div class="paragraph">
+            <p class="paragraph-text">Join us for live music, food trucks and much more fun for all the family. Featuring one of the most popular bands in Quito, Ecuador VPN!. See you there and lets rock!</p>
+        </div>
+        <div class="date-place">
+            <p class="date">2022.02.10(THU) ~ 11(FRI)</p>
+            <p class="place">@Coliseo General Rumiñahui, Quito EC</p>
+        </div>
+    </div>
+
+<!-- -----------------------PROGRAM--------------------------------- --> 
+
+    <div class="main-program">
+        <div class="main-p-title">
+            <h2 class="program-title">Main Program</h2>
+            <p class="orange-bar">_______</p>
+        </div>
+        <div class="cards-container">
```

Comment:
- [ ] In the Desktop version, please add the `: hover` action as shown in the image above, when the block is covered, make a white border around it.
- [ ] same thing for the image icon, please change them to something more similar to the original design

<img width="334" alt="icon bder" src="https://user-images.githubusercontent.com/79058364/157562383-35ac482f-8a81-4cc4-8dd3-05662dbb2e28.png">